Fix for getting clean builds with an updated libgtop

not suitable for upstream.

--- a/src/application.h	2017-03-20 23:34:16.000000000 -0700
+++ b/src/application.h	2017-11-29 13:25:15.751100782 -0800
@@ -3,6 +3,7 @@
 #define _GSM_APPLICATION_H_
 
 #include <gtkmm.h>
+#include "glibtop_machine.h"
 #include <glibtop/cpu.h>
 
 #include <algorithm>
